From f98c62b9edabeb770563978c0ed0fac4791901f8 Mon Sep 17 00:00:00 2001
From: Hoang Tran <hoang.tran@greenwavereality.com>
Date: Sun, 5 Oct 2014 23:31:24 +0800
Subject: [PATCH] fix json-c link

Signed-off-by: Hoang Tran <hoang.tran@greenwavereality.com>
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e8d0538..bade97c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,7 +13,7 @@ IF(APPLE)
 ENDIF()
 
 INCLUDE(FindPkgConfig)
-PKG_CHECK_MODULES(JSONC json-c json)
+PKG_CHECK_MODULES(JSONC json-c)
 IF(JSONC_FOUND)
   ADD_DEFINITIONS(-DJSONC)
   INCLUDE_DIRECTORIES(${JSONC_INCLUDE_DIRS})
@@ -45,7 +45,7 @@ INSTALL(TARGETS ubox ubox-static
 
 ADD_SUBDIRECTORY(lua)
 
-find_library(json NAMES json-c json)
+find_library(json NAMES json-c)
 IF(EXISTS ${json})
 	ADD_LIBRARY(blobmsg_json SHARED blobmsg_json.c)
 	TARGET_LINK_LIBRARIES(blobmsg_json ubox ${json})
-- 
2.1.1

